Client/Application/AppDelegate.swift

```diff
@@ -35,6 +36,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                      launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         log.info("startApplication begin")
 
+        NotificationCenter.default.addObserver(forName: .nimbusExperimentsApplied, object: nil, queue: nil) { _ in
+            self.runAppServicesHistoryMigration()
+        }
+
```

Comment:
We have something in our AppLaunchUtil, does adding it there help solve the purpose? 
This will allow us to not have bloated AppDelegate

https://github.com/mozilla-mobile/firefox-ios/blob/main/Client/Application/AppLaunchUtil.swift#L120-L127

Reply:
That's a lot cleaner thank you!